test/commands/members/index.js

```diff
@@ -34,7 +34,7 @@ b@heroku.com  collaborator
     .then(() => api.done());
   });
 
-  it('shows member when role is viewer', () => {
+  it('does not show member when role is viewer', () => {
```

Comment:
Why do we have a test here checking for viewer role, when we're now only returning member as a role?